crates/ironrdp-pdu/src/rdp/vc/dvc/ainput.rs

```diff
@@ -0,0 +1,259 @@
+use bitflags::bitflags;
```

Comment:
It would be great to add a link to FreeRDP docs for AInput PDUs structures

Reply:
ok